lib/talon/concern.ex

```diff
@@ -107,7 +107,7 @@ defmodule Talon.Concern do
 
       def page_names, do: @__page_map__ |> Map.keys
 
-      def dashboard(), do: List.first pages()
+      def dashboard(), do: List.first pages()  # TODO: default_page a better name? See #69 (DJS)
```

Comment:
default_page would be much better. I don't think we show have anything in talon hard coded to dashboard, except of the default page in the generator.